sig-contributor-experience/triage-team/triage.md

```diff
@@ -0,0 +1,43 @@
+# Issue and PR Management
+
+## Purpose
+
+Speed up management of issues and PRs labeled with `sig/contributor-experience`.
+
+The SIG Contributor Experience [issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience) and [PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+label%3Asig%2Fcontributor-experience+) are labeled with `sig/contributor-experience`.
+
+Following are few example searches on issue and PR:
+* [Open issues](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience)
+* [Open issues for a specific milestone](https://github.com/kubernetes/community/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Asig%2Fcontributor-experience+milestone%3AMay)
+* [Open PRs](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3A+pr+is%3Aopen+label%3Asig%2Fcontributor-experience+) 
+* [Open PRs for a specific milestone](https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Asig%2Fcontributor-experience+milestone%3AMay+is%3Aopen)
```

Comment:
For PRs, can we also add a note to assign approvers if the PR already has lgtm but no approvers are already assigned?

Reply:
@nikhita isn't that when lgtm is added, assignee gets created? That's what I see in multiple repos. I have created this query to test -  https://github.com/kubernetes/community/pulls?utf8=%E2%9C%93&q=is%3Apr+no%3Aassignee+label%3Asig%2Fcontributor-experience+label%3Algtm+is%3Aopen+
Please provide me a query that you have thought of? Thanks!

Reply:
I can't think of a github query you could use for this:
- has label:lgtm -label:approved
- read list of as-yet-unapproved OWNERS files in a comment on the PR
- get approvers out of that list
- see if there is someone in that list in the PR's assignees
- if not, add them

Reply:
Cool, thanks @spiffxp